course_catalog/migrations/0047_add_video_resource.py

```diff
@@ -0,0 +1,57 @@
+# Generated by Django 2.1.11 on 2019-10-21 19:24
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("course_catalog", "0046_add_learning_resource_offeror")]
+
+    operations = [
+        migrations.CreateModel(
+            name="VideoResource",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("created_on", models.DateTimeField(auto_now_add=True)),
+                ("updated_on", models.DateTimeField(auto_now=True)),
+                ("title", models.CharField(max_length=256)),
+                ("short_description", models.TextField(blank=True, null=True)),
+                (
+                    "_deprecated_offered_by",
+                    models.CharField(
+                        blank=True, db_column="offered_by", max_length=128, null=True
+                    ),
+                ),
+                ("video_id", models.CharField(max_length=80)),
+                ("platform", models.CharField(max_length=128)),
+                ("full_description", models.TextField(blank=True, null=True)),
+                ("image_src", models.URLField(blank=True, max_length=400, null=True)),
+                ("last_modified", models.DateTimeField(blank=True, null=True)),
+                ("published", models.BooleanField(default=True)),
```

Comment:
Do we need published? It should always be true since we are not going to get seperate api keys for each channel to get private videos

Reply:
This will cover the case of videos being removed, we have a practice of not deleting the records to maintain the integrity of user lists and learning paths, but `published=False` will mean that the item doesn't get included in the search index. We use this for courses that are no longer published since there's no value in searching for a course you can't take.

Reply:
Ok! That makes sense